jenkins/pipelines/android/Jenkinsfile

```diff
@@ -24,7 +24,7 @@ pipeline {
                 echo "Run Android Tests"
                 sh """
                     #export ANDROID_SERIAL=14151JEC204776
-                    jenkins/pipelines/android/android_tests.sh ${params.CBL_EDITION} ${params.CBL_VERSION} ${params.CBL_BUILD}
+                    jenkins/pipelines/android/android_tests.sh ${params.CBL_VERSION} ${params.CBL_BUILD} ${params.SGW_URL}
```

Comment:
As SGW_URL may contain `<ARCH>` which will get replaced magically with the right ARCH in a script used by SG docker, the `${params.SGW_URL}` should be used with quotes (single quotes is fine) here to escape '<' and '>' character.

```
sh """
jenkins/pipelines/android/android_tests.sh ${params.CBL_VERSION} ${params.CBL_BUILD} '${params.SGW_URL}'
"""
```

Reply:
oh!  Good catch!